### ISSUE TITLE
FFmpeg hangs when with proxychain

### ISSUE THREAD

**Issue by Deshdeepak1** [rofl0r/proxychains-ng#565]
Without proxychains both wget ffpmpeg properly opens the url
![image](https://github.com/rofl0r/proxychains-ng/assets/36617998/4a9ab978-745c-4a20-858c-f24f347f2491)

With proxychains only wget  properly opens the url, ffmpeg hangs
![image](https://github.com/rofl0r/proxychains-ng/assets/36617998/11db5545-7520-4627-aafe-cd4a1cd1883a)

Version: proxychains-ng 4.17-2

**Comment by rofl0r** [comment]
try running without -q so one can see what actually happens ?

**Comment by Deshdeepak1** [comment]
Still same, just hangs
![image](https://github.com/rofl0r/proxychains-ng/assets/36617998/79f44a1a-99be-41b7-b030-a4920fedea5d)

Works for wget 
![image](https://github.com/rofl0r/proxychains-ng/assets/36617998/e3607e1c-0847-4afe-8dcf-5c444890f00b)

**Comment by rofl0r** [comment]
i just tested it on ubuntu 22.04, works fine, also works on my own pc. so i guess the build you have does something that's not compatible. maybe try to get an ffmpeg build without cuda support.

**Comment by Deshdeepak1** [comment]
yeah, I also checked on my ubuntu vps , and it works. Something fishy about the arch build.
Let me check how to get ffmpeg without cuda support
